app/lib/rss/serializer.rb

```diff
@@ -9,7 +11,7 @@ def render_statuses(builder, statuses)
         item.title(status_title(status))
             .link(ActivityPub::TagManager.instance.url_for(status))
             .pub_date(status.created_at)
-            .description(status.spoiler_text.presence || Formatter.instance.format(status, inline_poll_options: true).to_str)
+            .description(status_description(status))
```

Comment:
I think this might lose the `RT @foo` bit when the toot is a boost.

Reply:
RSS feeds exclude reblogs so the `RT @foo` was never actually in use.